bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/calculatedValue.js

```diff
@@ -52,7 +52,8 @@ pimcore.object.classes.data.calculatedValue = Class.create(pimcore.object.classe
                 labelWidth: 140,
                 store: [
                     ['input', t('input')],
-                    ['textarea', t('textarea')]
+                    ['textarea', t('textarea')],
+                    ['html', t('HTML')]
```

Comment:
shouldn't this be lower cased? at least I couldn't find it in the master translation file.

Reply:
Oh sorry, you are right. Fixed it.